test/vaadin-dialog_draggable-resizable-test.html

```diff
@@ -263,12 +276,44 @@
 
         it('should resize content part when the overlay is resized', () => {
           resize(overlayPart.querySelector('.w'), -dx, 0);
-          const resizedBounds = overlayPart.getBoundingClientRect();
+
+          return waitUntil(() => {
```

Comment:
BTW is this syntax (returning Promise from `waitUntil`) guaranteed to work without making test function `async`?

Reply:
Cleaned up the new async logic in the tests and added a comment saying it's needed due to iOS 10 Safari. Using async required me to change eslint rules though, wdyt?

Reply:
Thanks, looks much cleaner now 👍